subsys/mgmt/mcumgr/Kconfig

```diff
@@ -279,6 +279,32 @@ config MCUMGR_SMP_UART
 	  commands to be received over UART without requiring an additional
 	  thread.
 
+if UART_ASYNC_API && MCUMGR_SMP_UART
+
+menuconfig MCUMGR_SMP_UART_ASYNC
+	bool "Use async UART API when available"
+	help
+	  The option enables use of UART async API when available for selected mcumgr uart port.
+
+if MCUMGR_SMP_UART_ASYNC
+config MCUMGR_SMP_UART_ASYNC_BUFS
+	int "Number of async RX UART buffers"
+	default 2
```

Comment:
maybe add range from 2, then comment about need for at least two will not be required.

Reply:
Done